backend/app/api/endpoints/kind/skills.py

```
@@ -204,3 +259,176 @@ def delete_skill(
+
+# ============================================================================
+# Unified Skill Endpoints (User + Public)
+# ============================================================================
+
+
+@router.get("/unified", response_model=List[UnifiedSkillResponse])
+def list_unified_skills(
+    skip: int = Query(0, ge=0, description="Number of items to skip"),
+    limit: int = Query(100, ge=1, le=100, description="Number of items to return"),
+    namespace: str = Query("default", description="Namespace filter"),
+    current_user: User = Depends(security.get_current_user),
```

Ruff (0.14.10)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)